### PR TITLE
docs: fix broken links in the monitoring section

### PR DESCRIPTION
**Pull request by NiccoloFei**

### CODE REVIEW

File: docs/src/monitoring.md [hunk 1]
```diff
@@ -359,7 +359,7 @@ go_threads 18
 ### User defined metrics
 
 This feature is currently in *beta* state and the format is inspired by the
-[queries.yaml file](https://github.com/prometheus-community/postgres_exporter/blob/master/queries.yaml) <!-- wokeignore:rule=master -->
+[queries.yaml file](https://github.com/prometheus-community/postgres_exporter/blob/v0.12.1/queries.yaml)
```
Comment (jsilvela) [inline docs/src/monitoring.md:362]: why specifically 0.12.1? There seem to be more recent versions.

Comment (jsilvela) [reply docs/src/monitoring.md:362]: OK, answered my own question. The more modern versions have no content 🤔

Comment (jsilvela) [inline docs/src/monitoring.md:362]: Um. Perhaps worth adding a note like `(deprecated)` or an explicit mention of the version?

Comment (jsilvela) [inline docs/src/monitoring.md:362]:
```suggestion
[queries.yaml file (release 0.12)](https://github.com/prometheus-community/postgres_exporter/blob/v0.12.1/queries.yaml)
```

Comment (jsilvela) [reply docs/src/monitoring.md:362]: wdyth @NiccoloFei ?

Comment (gbartolini) [reply docs/src/monitoring.md:362]: Wow ... ultimately the Prometheus exporter has adopted our preferred way to write queries in Go directly. Look:

https://github.com/prometheus-community/postgres_exporter/pull/801#issuecomment-1588258934

Comment (gbartolini) [reply docs/src/monitoring.md:362]: A new project has been created for this purpose, and database independent: https://github.com/burningalchemist/sql_exporter

**Review by jsilvela — APPROVED** [review]

**Review by sxd — APPROVED** [review]

**Pull request merged**